bodhi/server/static/js/update_form.js

```diff
@@ -70,22 +70,14 @@ $(document).ready(function() {
         },
     });
 
-    // candidate_error and bug_error are just two handy utilities for reporting
-    // errors when stuff in the code blocks below this goes wrong.
-    var candidate_error = function(package) {
-        $("#candidate-checkboxes .spinner").remove();
-        messenger.post({
-            message: 'No candidate builds found for ' + package,
-            type: 'error',
-        });
-    }
-    var bugs_error = function(package) {
-        $("#bugs-checkboxes .spinner").remove();
-        messenger.post({
-            message: 'No bugs found for ' + package,
-            type: 'error',
-        });
-    }
+    // Do not submit form when pressing enter while typing packages
+    // Instead select the first typeahead suggestion
+    $('#packages-search .typeahead').keydown(function(event) {
+        if(event.keyCode == 13) {
+            event.preventDefault();
+            $(".tt-selectable").first().click();
```

Comment:
This leads to a bit of an unexpected behavior if the user uses the arrow keys to scroll down to a different entry and then pressed enter. For example, if I search for "kernel" and then arrow down to something that isn't the first entry and press enter, it searches for the first entry.